### PR TITLE
Add possibility to include Mixpanel configs

### PR DESCRIPTION
**Pull request by thundron**
Change-type: major
Signed-off-by: Lorenzo Alberto Maria Ambrosi <lorenzothunder.ambrosi@gmail.com>

### CODE REVIEW

File: src/mixpanel.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ module.exports = (MixpanelLib) => {
      *   version: '1.0.0'
      * });
      */
-    install: (token, config) => {
+    install: (token, context, config) => {
```
Comment (zvin) [inline src/mixpanel.js:62]: This is a major change.

File: src/mixpanel.js [hunk 2]
```diff
@@ -111,9 +112,7 @@ module.exports = (MixpanelLib) => {
       const context = Object.assign({}, properties.context, utils.flattenStartCase(data))
 
       if (!properties.client) {
-        properties.client = MixpanelLib.init(properties.projectToken, {
-          protocol: 'https'
-        }) || MixpanelLib
+        properties.client = properties.client = MixpanelLib.init(properties.projectToken, properties.config) || MixpanelLib
```
Comment (zvin) [inline src/mixpanel.js]: properties.client = properties.client = ...

**Comment by thundron** [comment]
@resin-ci retest

**Review by zvin — CHANGES_REQUESTED** [review]

File: src/mixpanel.js (hunk 1 above)
Comment (thundron) [reply src/mixpanel.js:62]: Right, I'll tag the commit appropriately

File: src/mixpanel.js (hunk 2 above)
Comment (thundron) [reply src/mixpanel.js]: :/ my bad, copy-paste leftover and didn't catch it because in the last test (where I copied that) I didn't enter that conditional

**Review by zvin — APPROVED** [review]

**Pull request merged**